data-prepper-logstash-configuration/src/main/resources/org/opensearch/dataprepper/logstash/mapping/kv.mapping.yaml

```diff
@@ -0,0 +1,11 @@
+pluginName: key_value
+mappedAttributeNames:
+  source: source
```

Comment:
FYI, the converter should already support attributes with the same name in Logstash as in Data Prepper. You can still use this, but it isn't necessary.